app/src/main/java/edu/uc/groupProject/topten/MainActivity.kt

```diff
@@ -37,89 +23,6 @@ class MainActivity : AppCompatActivity() {
         }
 
         this.supportActionBar?.hide();
```

Comment:
Moved all database logic to MainViewModel.
As I understand the class material to mean that it should be centralized and provided by a ViewModel.